frontend/playwright.config.ts

```diff
@@ -114,8 +116,8 @@ const config: PlaywrightTestConfig = {
 
    /* Run your local dev server before starting the tests */
    webServer: {
-      cwd: '../',
-      command: 'pnpm run dev',
+      cwd: process.env.CI ? './' : '../',
```

Comment:
What's this change about?

Reply:
In CI, we want to run only the `frontend` (in the current `./` frontend folder) because the `backend` will be already running using docker.

However, when testing locally, we want both back and front ends to start so we have to be in the root folder (thus, we use `../`).